qa-tests-backend/scripts/workload-identities/workload-identities.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Workload identity resources for Central.
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../../.. && pwd)"
+# shellcheck source=../../../scripts/lib.sh
+source "$ROOT/scripts/lib.sh"
+# shellcheck source=../../../scripts/ci/gcp.sh
+source "$ROOT/scripts/ci/gcp.sh"
+
+set -euo pipefail
+
+setup_workload_identities() {
+    if [[ "${SETUP_WORKLOAD_IDENTITIES:-false}" == "false" ]]; then
+        info "Skipping the workload identity setup."
+        return 0
+    fi
+    setup_gcp_workload_identities
+}
+
+cleanup_workload_identities() {
+    if [[ "${SETUP_WORKLOAD_IDENTITIES:-false}" == "false" ]]; then
+        info "Skipping the workload identity cleanup."
+        return 0
+    fi
+    cleanup_gcp_workload_identities
+}
+
+setup_gcp_variables() {
+    cluster=$(kubectl config view --minify -o jsonpath="{.clusters[].name}")
+    service_account="stackrox-ci-workload-identity@acs-san-stackroxci.iam.gserviceaccount.com"
+    project="280228816191" # acs-san-stackroxci
```

Comment:
I think these should go in the CI vault but I don't think you have access: https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/stackrox-stackrox-e2e-tests/credentials

Reply:
nvmd, I see you are in the list 'shesselm'.